cli/src/main/java/org/opentosca/toscana/cli/ApiController.java

```diff
@@ -484,109 +313,115 @@ public String inputList(String csar, String plat) throws IOException {
-            if (plat.getId() != null) {
-                return con.PLATFORM_INFO_SUCCESS + plat.getId() + "\nName: " + plat.getName();
-            } else {
-                return con.PLATFORM_INFO_EMPTY;
-            }
-        } else if (response.code() == 404) {
-            if (response.errorBody().string() != null) {
-                return con.PLATFORM_INFO_ERROR404 + "\n" + response.errorBody().string();
-            } else {
-                return con.PLATFORM_INFO_ERROR404;
-            }
-        } else {
-            return con.PLATFORM_INFO_ERROR;
+    public String infoPlatform(String plat) {
+        Platform platform;
+        StringBuilder stringPlatform = new StringBuilder();
+        try {
+            platform = toscAnaAPI.getPlatformDetails(plat);
+            stringPlatform.append(platform.getId()).append(", ").append(platform.getName());
+        } catch (IOException e) {
+            System.err.println(String.format(SOMETHING_WRONG + Constants.PLATFORM_INFO_ERROR + " '%s'", e.getMessage()));
+            e.printStackTrace();
+        } catch (TOSCAnaServerException e) {
+            System.err.println(String.format(SOMETHING_WRONG + Constants.PLATFORM_INFO_ERROR + " %s '%s'", e.getStatusCode(), e.getErrorResponse().getMessage()));
         }
+        return stringPlatform.toString();
     }
 
     /**
      Calls the REST API and returns the current state of the system
 
      @return output for the CLI
-     @throws IOException if the responsebody is null
      */
-    public String showStatus() throws IOException {
-        Call<Status> statusCall = service.getSystemStatus();
-        Response<Status> response = statusCall.execute();
-        Status status = response.body();
-
-        if (response.code() == 200) {
-            if (status.getStatus() != null) {
-                return con.STATUS_SUCCESS + "\nStatus: " + status.getStatus()
-                    + "\nAvailable Storage: " + status.getAvailableStorage()
-                    + "\nTotal Storage: " + status.getTotalStorage();
-            } else {
-                return con.STATUS_EMPTY;
-            }
-        } else {
-            return con.STATUS_ERROR;
+    public String showStatus() {
+        TransformerStatus status;
+        StringBuilder stringStatus = new StringBuilder();
+        try {
+            status = toscAnaAPI.getServerStatus();
+
+            stringStatus.append(status.getStatus()).append(", free: ").append(status.getFileSystemHealth().getFreeBytes());
+        } catch (IOException e) {
+            System.err.println(String.format(SOMETHING_WRONG + Constants.STATUS_ERROR + " '%s'", e.getMessage()));
+            e.printStackTrace();
+        } catch (TOSCAnaServerException e) {
+            System.err.println(String.format(SOMETHING_WRONG + Constants.STATUS_ERROR + " %s '%s'", e.getStatusCode(), e.getErrorResponse().getMessage()));
+        }
+        return stringStatus.toString();
+    }
+
+    public String showMetrics() {
```

Comment:
I honestly dont think the CLI has to mess with the metrics query, it is horrible to read and horrible to create a pretty output. 

I would just consider ditching this functionality. Or you take a look at the map and find useful information that could be displayed such as the uptime, the heapspace used...